test/test_linalg.py

```diff
@@ -1873,19 +1873,6 @@ def run_subtest(actual_rank, matrix_size, batches, device, svd_lowrank, **option
         actual_rank, size, batches = 2, (17, 4), ()
         run_subtest(actual_rank, size, batches, device, jitted)
 
-    @onlyCPU
-    @skipCPUIfNoLapack
-    @dtypes(torch.cfloat)
-    def test_svd_complex(self, device, dtype):
```

Comment:
Should this test be repurposed to verify the new behavior?

Reply:
Yes, it should. I'll do that.